internal/logger/logger.go

```diff
@@ -35,7 +35,7 @@ func init() {
 	log.SetReportCaller(false)
 
 	log.Formatter = &formatter.Formatter{
-		TimestampFormat: time.RFC3339,
+		TimestampFormat: time.RFC3339NanoNano,
```

Comment:
should be `RFC3339Nano`

Reply:
> should be `RFC3339Nano`

yeah sorry it's my bad as I changed them by sed command. I already modify it and send request again. tnx